### PR TITLE
Fix language file not loading and remove unnecessary translate keys

### PR DESCRIPTION
**Pull request by misode**

### CODE REVIEW

**Comment by AndanteDevs** [comment]
Could you also remove the `Bat Droppings` key in this pull?

**Comment by misode** [comment]
> Could you also remove the `Bat Droppings` key in this pull?

Yep, I'll also make a few extra changes

File: GM4 Resources/assets/gm4/lang/en_us.json [hunk 1]
```diff
@@ -15,6 +15,10 @@
     "text.gm4.modules.update.installing": "Installing %s...",
     "text.gm4.modules.update.installed": "%s Installed!",
 
+    "text.gm4.xp_storage.stored_experience": "Stored Experience: ",
+    "text.gm4.xp_storage.stored_experience.points": "%s Points",
+    "text.gm4.zauber_cauldrons.wormhole.offhand_unsupported": "Offhand not supported in versions below 1.14!",
```
[the remark this answers is not in the document]
Comment (misode) [reply GM4 Resources/assets/gm4/lang/en_us.json]: Nope, I already removed this locally, I will push soon

**Comment by misode** [comment]
This is now ready to be merged.

**Pull request merged**